web/client/components/data/featuregrid/toolbars/Toolbar.jsx

```diff
@@ -16,19 +16,19 @@ const getSaveMessageId = ({saving, saved}) => {
     return "featuregrid.toolbar.saveChanges";
 };
 
-module.exports = ({events = {}, mode = "VIEW", selectedCount, hasChanges, hasGeometry, hasNewFeatures, isSimpleGeom, isDrawing = false, isEditingAllowed, saving = false, saved = false, isDownloadOpen, isColumnsOpen, disableToolbar} = {}) =>
+module.exports = ({events = {}, mode = "VIEW", selectedCount, hasChanges, hasGeometry, hasNewFeatures, isSimpleGeom, isDrawing = false, isEditingAllowed, saving = false, saved = false, isDownloadOpen, isColumnsOpen, disableToolbar, isCesium} = {}) =>
     (<ButtonGroup id="featuregrid-toolbar" className="featuregrid-toolbar featuregrid-toolbar-margin">
         <TButton
             id="edit-mode"
             tooltip={<Message msgId="featuregrid.toolbar.editMode"/>}
-            disabled={disableToolbar}
+            disabled={disableToolbar || isCesium}
```

Comment:
Components do not have to be aware of cesium. disableToolbar is enough, set it accordingly in the plugin